sidebars.js

```diff
@@ -55,6 +55,15 @@ module.exports = {
                     'api/system'
                 ]
             },
+            {
+                type: 'category',
+                label: 'Metrics',
```

Comment:
It's best to label "Metrics for Prometheus", otherwise new users may not know what they're for.